cmd/banner.go

```diff
@@ -130,4 +163,5 @@ var bannerCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(bannerCmd)
+	bannerCmd.Flags().Bool("no-wait-for-supervisor", false, "Don't wait until supervisor is started")
```

Comment:
I think that is generally okay, let's simplify the option `--no-wait`. Maybe we can later move that on top level of the CLI